modules/openapi-generator/src/main/resources/dart2/api_exception.mustache

```diff
@@ -6,9 +6,9 @@ class ApiException implements Exception {
   ApiException.withInner(this.code, this.message, this.innerException, this.stackTrace);
 
   int code = 0;
-  String message;
-  Exception innerException;
-  StackTrace stackTrace;
+  String? message;
```

Comment:
another design question, when it makes sense to raise ApiException without a message?